x-pack/plugins/cases/public/components/all_cases/all_cases_list.test.tsx

```diff
@@ -101,15 +100,50 @@ describe.skip('AllCasesListGeneric', () => {
   };
 
   const removeMsFromDate = (value: string) => moment(value).format('YYYY-MM-DDTHH:mm:ss[Z]');
+  // eslint-disable-next-line prefer-object-spread
+  const originalGetComputedStyle = Object.assign({}, window.getComputedStyle);
 
   let appMockRenderer: AppMockRenderer;
 
   beforeAll(() => {
+    // The JSDOM implementation is too slow
+    // Especially for dropdowns that try to position themselves
+    // perf issue - https://github.com/jsdom/jsdom/issues/3234
+    Object.defineProperty(window, 'getComputedStyle', {
```

Comment:
Great finding!! 👏  
perhaps we could try the same for `xpack/plugins/cases/public/components/case_view/components/
case_view_activity.test.tsx` component as well.

Reply:
@jcger told me about it! Yes I try to improve the `case_view_activity.test.tsx` on another PR.